scripts/medistat.shader

```diff
@@ -39,6 +54,7 @@ models/buildables/medistat/display
 		blendfunc add
 	}
 	when idle2 models/buildables/medistat/displayH
+	when unpowered textures/shared_colors/black
```

Comment:
```suggestion
when unpowered models/buildables/arm/screen
```

Well, I'd put this image into a generic shader too, ofc, but the black texture is really ugly.

Reply:
Ok then.